README.md

```diff
@@ -148,7 +149,7 @@ To run the project, execute the following command:
 > Describe 1 - 3 features you will add to the project.
```

Comment:
**[Optional]**

- [ ] It would be nice if you remove this section description to make your readme file look clean and professional 😊 Also this is just a instruction for you not for the all the viewer of your repository 🤔

Reply:
Hi @HAMMAS-SALEEM Thanks for the observation. I have edited my readme file. 😃